Assets/_Application/Layers/LayerTypes/Polygons/PolygonInputToLayer.cs

```diff
@@ -220,7 +220,7 @@ private void CreateLineLayer(List<Vector3> unityLine)
                     layers.Add(polygonSelectionLayer.PolygonVisualisation, polygonSelectionLayer);
                     polygonSelectionLayer.polygonSelected.AddListener(ProcessPolygonSelection);
                     lineInput.SetDrawMode(PolygonInput.DrawMode.Edit); //set the mode to edit explicitly, so the reselect functionality of ProcessPolygonSelection() will immediately work
-                    ProcessPolygonSelection(polygonSelectionLayer);
+                    ProcessPolygonSelection(polygonSelectionLayer); //TODO is this still needed?
```

Comment:
dit is zodat je polygon opnieuw geselecteerd wordt na de edit (zodat je verder kan editen) als dat nog kan zonder deze line dan kan dat weg